docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -77,6 +77,14 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
+* Edit /etc/kubernetes/controller-manager to add fed-node to the
+  KUBELET_ADDRESSES.
+
+```
+# Comma separated list of minions
+KUBELET_ADDRESSES="--machines=fed-node"
```

Comment:
We just switched away from this method.  It stinks because it doesn't let you grow/shrink your cluster.  Why do you want to go back?